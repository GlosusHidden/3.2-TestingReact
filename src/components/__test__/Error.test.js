import React from 'react'
import renderer from 'react-test-renderer'
import { Provider } from 'react-redux'

import Error from '../Error'

import {myStore, mockStore} from './test_data/data'


it('Error Component Snapshot', () => {
  const renderedValue = renderer.create(
    <Provider store={mockStore}>
      <Error />
    </Provider>
  ).toJSON();
  expect(renderedValue).toMatchSnapshot();
});
