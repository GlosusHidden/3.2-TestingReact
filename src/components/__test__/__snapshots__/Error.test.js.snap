// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error Component Snapshot 1`] = `
<div
  className="placeholder"
>
  <div
    className="text"
  >
    Error, try again later..
  </div>
  <img
    alt="loading"
    className="placeholder_img"
    src="img/error.gif"
  />
</div>
`;
